lib/active_record/sql_analyzer/compact_logger.rb

```diff
@@ -13,13 +13,14 @@ def initialize(*)
       end
 
       def log(event)
-        sha = Digest::MD5.hexdigest(event.to_s)
```

Comment:
@nerdrew This file has the actual relevant changes - the rest is Rubocop fixups.